### DISCUSSION TITLE
How to pip install old version of this library?

### DISCUSSION THREAD

**Discussion by akarri2001** [ramnes/notion-sdk-py#45] [Q&A] [answered]
Hi!
One of the scripts that I made uses the 0.3.1 version of this library but does not work well with the current version of the library. Is there a way to pip install that version of the library instead?

**Comment by ramnes** [comment, marked as answer]
`pip install notion-client==0.3.1` :)

**Reply by aahnik** [reply]
Hi, @akarri2001 or @ramnes please mark this as answered.

**Reply by akarri2001** [reply]
Thank you! I just figured out last night that the query requests were directly returning a dictionary, so I managed to update the code to match the current version of the SDK

**Reply by ramnes** [reply]
That's even better! We should probably add a changelog by the way, so let's open an issue for that. :)

**Reply by ramnes** [reply]
#48
